openapi.yml

```diff
@@ -1263,8 +1291,8 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/DescriptiveValue"
-        source:
-          $ref: "#/components/schemas/Source"
         version:
-          description: The version of the standard or encoding.
+          description: The version of the value standard.
```

Comment:
What is a value standard?

Reply:
Value standard is about how you derive and construct the metadata values. For example, MARC says the title goes in 245$a, but it's the content standard RDA that has rules about where you get the title from (cover? title page? spine?), how you transcribe it, what kinds of title variations should be included, how to handle a typo on the resource, etc. Other examples of library value standards are DACS (archives) and the various RBMS standards (rare books and manuscripts).

Reply:
Can you add `e.g., RDA`? At least for me, that would make it much clearer.